service/topology_coordinator.cc

```diff
@@ -880,6 +880,10 @@ class topology_coordinator : public endpoint_lifecycle_subscriber {
         return _topo_sm._topology.get_excluded_nodes().contains(server_id);
     }
 
+    bool is_ignored(raft::server_id server_id) {
```

Comment:
Why can't we use is_excluded() instead of the new is_ignored()? What's the semantic difference? get_excluded_nodes() currently just returns ignored_nodes.

Reply:
Yes, you're right, they're the same. Will fix in v2